### PR TITLE
Update find command

### LINKED ISSUE #112: Update find student command to be able to find appointments
Find appointments will work hand in hand with find students.  Finding a student will also show the relevant appointments that has been assigned to the student.

### PR DESCRIPTION
**Pull request by SiTingST**
Upgrade find command to finding both the student and appointment entries with a particular student's matriculation number

### CODE REVIEW

**Review by fairyinabottle4 — APPROVED** [review]
LGTM

File: src/main/java/seedu/student/model/Model.java [hunk 1]
```diff
@@ -15,8 +15,9 @@
 public interface Model {
     /** {@code Predicate} that always evaluate to true */
     Predicate<Student> PREDICATE_SHOW_ALL_STUDENTS = unused -> true;
-
     /**
+     *
```
Comment (yienyoong) [inline src/main/java/seedu/student/model/Model.java:19]: You may want to consider removing these extra blank comment lines :)

File: src/test/java/seedu/student/logic/commands/AddAppointmentCommandTest.java [hunk 2]
```diff
@@ -159,6 +159,11 @@ public void updateFilteredStudentList(Predicate<Student> predicate) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public void updateFilteredAppointmentList(Predicate<SameDateAppointmentList> predicate) {
+            throw new AssertionError("This method should not be called.");
+        }
+
```
Comment (yienyoong) [inline src/test/java/seedu/student/logic/commands/AddAppointmentCommandTest.java:166]: Nice use of assertions!

**Review by yienyoong — APPROVED** [review]
Very good work. Thank you Si Ting!

**Pull request merged**